src/rust/engine/src/rule_graph.rs

```diff
@@ -674,38 +674,30 @@ impl<'t> GraphMaker<'t> {
     if satisfiable_entries.is_empty() {
```

Comment:
Could these Rust changes be split out into a separate PR, so that this one becomes solely a doc update?